Proton/Sources/Editor/EditorView.swift

```diff
@@ -882,6 +882,17 @@ extension EditorView {
     }
 
     func relayoutAttachments(in range: NSRange? = nil) {
+        guard
+            richTextView.textContainer.size.height > 0,
+            richTextView.textContainer.size.height < CGFloat(Float.greatestFiniteMagnitude)
```

Comment:
Another interesting thing I found, `CGFloat.greatestFiniteMagnitude` is not the same value as `Float.greatestFiniteMagnitude`. The layout pass seems to want `Float` value... but as a CGFloat.

Reply:
`< CGFloat(Float.greatestFiniteMagnitude)` helped for TableView cell, but breaks some layouts when opening Editor in a Modal. The `> 0` seems to be safe.

I've declined this PR in favour of https://github.com/rajdeep/proton/pull/80